test/JDBC/input/BABEL-3201-vu-verify.sql

```diff
@@ -0,0 +1,35 @@
+DBCC CHECKIDENT(babel_3201_t1);
```

Comment:
Please try to add at least one test case for all the errors these changes are introducing.

Reply:
I have added test cases for all the functional scenarios, corner cases ( no rows have been inserted in table, rows are removed using TRUNCATE TABLE, rows are deleted using DELETE TABLE), and negative tests for syntax and datatype bounds.